Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml

```diff
@@ -1648,30 +1648,28 @@ tasks:
     task:
       id: 3ba84f9e-aaf6-4bd3-8d22-cfc00499e075
       version: -1
-      name: Should continue with the playbook's investigation and response?
+      name: Whether to continue with the investigation and response?
```

Comment:
I think we can rename the task to something like "check for unhandled alert" / "Check whether all alerts handled?" / etc

Reply:
Agreed. changed to "Check whether there are any unhandled alerts?"